packages/admin/cms-admin/src/common/header/UserHeaderItem.tsx

```diff
@@ -51,7 +58,12 @@ export function UserHeaderItem(props: UserHeaderItemProps): React.ReactElement {
     const { loading, data } = useQuery<GQLCurrentUserQuery>(currentUserQuery);
     const [signOut, { loading: isSigningOut }] = useMutation<GQLSignOutMutation>(signOutMutation);
 
-    if (loading || !data) return <CircularProgress />;
+    if (loading || !data)
+        return (
+            <LoadingWrapper>
```

Comment:
Loading wrappers shouldn't be needed in application code, they're fine in library code. Why do you need it here?

Reply:
Thanks for the explanation!

Should we even display a loading indicator here? 🤔

Reply:
It might assure the user that there is still something coming, when they want to click on the user-button and it takes a bit longer 🤷🏼

Reply:
Alright 👍🏼